### DISCUSSION TITLE
ScrollSpy - multiple 'active' nav items at once?

### DISCUSSION THREAD

**Discussion by itsViney** [twbs/bootstrap#36654] [Q&A] [answered]
I have a page where navigable sections (listed in the ScrollSpy nav) have the possibility to appear inline with each other on large enough screens. They're always stacked vertically on mobile, but on desktop two or more sections appear side by side. In this scenario, ScrollSpy only applies the 'active' class to whichever is last in the DOM.

Is there a way to have ScrollSpy apply the 'active' class to **all** matching nav items, if their targets all come into view at once?

**Comment by isugar21sum** [comment]
https://getbootstrap.com/docs/5.2/component/scrollspy/

Im pretty sure there is, id give you my idea. but sharing this might help

**Reply by itsViney** [reply]
I've read the docs and there's nothing in there that covers this.

**Comment by GeoSot** [comment, marked as answer]
Unfortunately scrollspy is made up to support only one visible element. 
Probably you can experiment on these lines and try to modify caclulation process based on a given option/flag
Of couser if you come up to a solution we will appreciate you to open a PR :smile: 
https://github.com/twbs/bootstrap/blob/c1813ef2bc2a398e5a9bcdd398608f8a021574e3/js/src/scrollspy.js#L150-L187
